examples/Sodium/fit_sodium_dft.jl

```diff
@@ -0,0 +1,83 @@
+using LinearAlgebra, Random, Statistics, StatsBase, Distributions
+using AtomsBase, Unitful, UnitfulAtomic, StaticArrays
+using InteratomicPotentials, InteratomicBasisPotentials
+using CairoMakie
+using JLD
+using DPP
+include("./src/PotentialLearning.jl")
+using .PotentialLearning
```

Comment:
Nitpick we should be doing just `using PotentialLearning` maybe needs a `push!(Base.LOAD_PATH, dirname(@__DIR___))`

Reply:
Good call